statvar_imports/gdp_by_county_metro_and_other_areas/beagdpv2/README.md

```diff
@@ -0,0 +1,42 @@
+# BEAGDPv2
+
+The data set contains GDP by  county, Metro, and Other Areas
+
+Download:
+Data download URL : `https://apps.bea.gov/regional/zip/CAGDP9.zip`
+Select the CAGDP9: Real GDP in chaied dollars by County & MSA
+
+
+Processing: 
+Earlier code : https://source.corp.google.com/piper///depot/google3/datacommons/mcf/bea/v2/ ( 2001-2021 data)
+Current execution : Using statvarProcessor ( 2001-2023 data).
+
+###File paths in gcs:
+
+input file: gs://unresolved_mcf/us_bea/gdp_chained_dollar_county_msa/latest/input_files/CAGDP9__ALL_AREAS_2001_2023.csv
+pv_map: gs://unresolved_mcf/us_bea/gdp_chained_dollar_county_msa/latest/input_files/pv_map.py
+place mappings : gs://unresolved_mcf/us_bea/gdp_chained_dollar_county_msa/latest/input_files/place_mapping.csv
+config file: gs://unresolved_mcf/us_bea/gdp_chained_dollar_county_msa/latest/input_files/bea_metadata.py
+output files: gs://unresolved_mcf/us_bea/gdp_chained_dollar_county_msa/latest/
+stat_vars.mcf : gs://unresolved_mcf/scripts/statvar/stat_vars.mcf
+
```

Comment:
Let's drop this since these files are in this folder now and don't need GCS files.

Also please note that the README is also for external users who may not have access to these GCS buckets.